### ISSUE TITLE
[Bug] Xamarin Form 4.5, 4.6 Shell.FlyoutBehavior="Disabled" not working on iOS

### ISSUE THREAD

**Issue by liweipro** [xamarin/Xamarin.Forms#10694]
### Description
With Xamarin Form 4.5 and 4.6, Shell.FlyoutBehavior="Disabled" not working on iOS. The flyout menu can still be brought out by swiping from left to right. This is a very important bug to us, as it allows unauthorized users to bring out the menu and gets access to certain features of the app.

### Steps to Reproduce

1. Set the behavior in code or XAML like  Shell.SetFlyoutBehavior(this, FlyoutBehavior.Disabled);

### Expected Behavior
On the page, the flyout menu should be disabled, and by swiping from left to right should not bring out the menu

### Actual Behavior
The menu can be brought out by swiping from left to right on iOS

### Basic Information

- Version with issue: 4.5, 4.6
- Last known good version: Xamarin Form 4.4
 - iOS:  tested on 13.4
 - Android: Android is fine. The behavior works as expected

**Comment by PureWeen** [comment]
@liweipro what type of device are you using?

Does this look like it'll fix your issue?
https://github.com/xamarin/Xamarin.Forms/pull/10632

This the hamburger still visible?

**Comment by liweipro** [comment]
> @liweipro what type of device are you using?
> 
> Does this look like it'll fix your issue?
> #10632
> 
> This the hamburger still visible?

hi @PureWeen , I have tested the 4.6 service release 2 on iOS, the issue has been fixed, thank you.

There is a minor issue. 

When the app starts, it shows a page with flyout menu disabled. The swipe gesture is disabled properly on iOS now. After the app later enters a page with flyout menu enabled. The swipe gesture is somehow not enabled until I either,
1. click the breadcrumb icon to bring out the menu
2. or enter a child page
By doing either one of the two actions above, the swipe gesture will start working properly again. I will attach an example later
